awot/io/read_sonde.py

```diff
@@ -81,14 +100,36 @@ def read_dropsonde_data(filePath, split_file=True):
     '''
     method to retrieve dropsonde data
 
-    Inputs:
-
-    file path
-    #============
-
-    output:
-
-    #Dictionary with variable for use in plotting
+    Parameters
+    -----------
+    filepath: string
+        long path file name
+        
+    splitfile: string
+        keyword string telling the method to split the file at header
+
+    output
+    ---------
+    data : dict
+        AWOT dictionary instance.
+        metadata : dict
+            Dictionary of global attributes in file.
+        temperature: dict
+            Temperature dictionary [c]
+        dewpoint: dict
+            Dewpoint dictionary [c]
+        pressure: dict 
+            pressure dictionary [hpa]
+        relative_humidity: dict 
+            RH dictionary
+        U_component : float
+            Wind along u axis wind [m/s].
+        V_component : float
+            Wind perpendicular to u axis wind [m/s].
+        height : dict
+            Height array [m].
+        data_format: string
+            format string radiosonde or dropsonde.
 

```

Comment:
Can you add periods to the labels that don't have them? Also RH should have [%] added. for consistency.